halo/app/monitor.go

```diff
@@ -236,8 +247,8 @@ func startMonitoringAPI(
 	}
 
 	go func() {
-		if err := server.ListenAndServe(); err != nil && err != http.ErrServerClosed {
```

Comment:
see `server.ListenAndServe` docs, it always returns a non-nil error, so no need to check for nil